docs/install-azure-repos.md

```diff
@@ -0,0 +1,202 @@
+[Go back to the main documentation page](../README.md)
+
+# Installing Frogbot on Azure Repos repositories
+
+To install Frogbot on Azure Repos repositories:
+
+1. Go to your Azure Pipelines project, and add a new pipeline:
+
+   ![azure-new-pipeline.png](../images/azure-new-pipeline.png)
+
+
+2. Set `Azure Repos Git` as your code source:
+
+   ![azure-set-code-source.png.png](../images/azure-set-code-source.png)
+
+
+3. Select the repository you'd like Frogbot to scan.
+
+   ![azure-select-repo-to-test.png](../images/azure-select-repo-to-test.png)
+
+
+4. Select `Starter Pipeline` and name it `frogbot-scan-pr.yml`. Use the content of the below yaml for the pipeline.
```

Comment:
... the below yaml for the pipeline. --> ... the below template for the pipeline.
The same goes for section 5 below.

Reply:
done